packages/suite/src/components/suite/modals/Pin.tsx

```diff
@@ -4,17 +4,15 @@ import { Translation, Modal, ModalProps } from '@suite-components';
 import { PinMatrix, PIN_MATRIX_MAX_WIDTH } from '@suite-components/PinMatrix';
 import { TrezorDevice } from '@suite-types';
 
-const StyledModal = styled(Modal)`
+const StyledModal = styled(Modal)<{ $isExtended: boolean }>`
```

Comment:
What is the dollar sign for here?

Reply:
A transient prop. It prevents the prop from being propagated to the component and only affects the style. I noticed Szymon used it elsewhere in the code. https://styled-components.com/docs/api#transient-props

Reply:
Cool!